src/topology/homotopy/pi_n.lean

```diff
@@ -8,22 +8,25 @@ import topology.homotopy.basic
 import topology.path_connected
 import topology.unit_interval
 import algebraic_topology.fundamental_groupoid.basic
+import logic.unique
 
 /-!
 # nth homotopy group
 
 We define the nth homotopy group at x `π n x` as the equivalence classes
 of functions from the nth dimensional cube to the topological space X
 that send the boundary to the base point `x`, up to homotopic equivalence.
+Note that such functions are nth dimensional loops `nloop n x`, in particular
+`nloop 1 x ≃ path x x`
 
 We show that `π 0 x` is equivalent to the path-conected components, and
 that `π 1 x` is equivalent to the fundamental grupoid at `x`.
 
 ## definitions
 
-* `pre_π n x` is the type of continous fuctions `I^n→X` that send the boundary to `x`
-* `pre_π.homotopy` is the type of homotopies between such functions
-* `π n x` the quotient of `pre_π n x` up to homotopic equivalence
+* `nloop n x` is the type of continous fuctions `I^n→X` that send the boundary to `x`
+* `nth_homotopy_group n x` denoted `π n x` is the quotient of `nloop n x` up to homotopic
```

Comment:
We don't usually include variable names in declaration names; `homotopy_group n x` should be good enough, but I'm not so sure about `loop` in root namespace.

Reply:
How about `gen_loop` for generalized loop?